src/integrations/prefect-gcp/prefect_gcp/bigquery.py

```diff
@@ -48,6 +42,7 @@ def _result_sync(func, *args, **kwargs):
 
 
 @task
+@sync_compatible
```

Comment:
these are to maintain the interface since async flows were imported and used as if they were sync due to 2.x engine mechanics